src/components/header.jsx

```diff
@@ -55,12 +55,11 @@ function Header({ slug }) {
   );
 
   const handleClick = (langKey) => {
-    i18n.changeLanguage(langKey);
-    navigate(`/${langKey}/${slug}`);
+    i18n.changeLanguage(langKey, navigate(`/${langKey}/${slug}`));
```

Comment:
Whaaaat, where'd you learn this?

Reply:
I found it [here](https://www.i18next.com/overview/api#changelanguage) when I was looking into the language direction method